src/TTextEdit.cpp

```diff
@@ -540,15 +536,17 @@ int TTextEdit::drawGrapheme(QPainter& painter, const QPoint& cursor, const QStri
         bgColor = charStyle.background();
     }
 
-    auto textRect = QRect(mFontWidth * cursor.x(), mFontHeight * cursor.y(), mFontWidth * charWidth, mFontHeight);
+    //auto metrics = QFontMetrics(painter.font());
+    auto advance = mFontMetrics.horizontalAdvance(grapheme);
```

Comment:
This is only available in Qt 5.11 while we support 5.7 at minimum - right now. However I'd like to up the mininum to 5.11 because Debian stable now supports it, so this shouldn't be a problem.

Reply:
Didn't know sorry, I'll check if I can achieve the same result using QFontMetrics::boundingRect

Reply:
Don't spend time on it yet, https://github.com/Mudlet/Mudlet/pull/2943/